conan/tools/gnu/gnudeps_flags.py

```diff
@@ -60,15 +59,14 @@ def _format_frameworks(self, frameworks, is_path=False):
     def _format_include_paths(self, include_paths):
         if not include_paths:
             return []
-        # FIXME: Missing support for subsystems
-        return ["-I%s" % (self._adjust_path(include_path))
+        pattern = "/I%s" if is_msvc(self._conanfile) else "-I%s"
```

Comment:
Why? In theory VS also admits ``-I`` without problems. Maybe it is a bit more compatible if this is used by some other build tool, like autotools + cl.exe?

Reply:
Yes, it admits both but Meson is failing because of an internal bug on its side. So this change is making life easier for Meson users and works with VS.

Reply:
if it's more compatible with tools (e.g. meson), let's switch